projects/control-service/projects/pipelines_control_service/src/main/java/com/vmware/taurus/service/execution/JobExecutionResultManager.java

```diff
@@ -118,12 +119,14 @@ private static ExecutionTerminationStatus updateTerminationStatusBasedOnExecutio
          ExecutionTerminationStatus terminationStatus,
          ExecutionStatus executionStatus,
          String terminationStatusString,
-         String terminationReason) {
+         String terminationReason,
```

Comment:
Maybe rename this to `jobTerminationReason` now that we have two different reasons.

Reply:
Done, thanks